packages/fast-exit/contracts/helpers/buddy-bridge/arbitrum/BuddyDeployer.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity  >=0.6.11;
+
+import "arbos-precompiles/arbos/builtin/ArbSys.sol";
+import "../ethereum/L1Buddy.sol";
+
+contract BuddyDeployer {
```

Comment:
I don't think these buddy bridge contracts are useful for anything anymore
we could delete them

Reply:
How about the L1Buddy which we use in the BatchMover contract?

Reply:
I think we can just remove that part of the logic tbh
the buddy deployer doesn't really help imo